oneanddone/users/tests/test_api.py

```diff
@@ -96,14 +97,18 @@ def test_delete_user(self):
         self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.token.key)
         test_user = UserFactory.create()
         user_uri = self.uri + test_user.email + '/'
+        get_response = User.objects.get(username=test_user.username)
 
         # Make a DELETE request
         delete_response = self.client.delete(user_uri)
         self.assert_response_status(delete_response, status.HTTP_204_NO_CONTENT)
 
         # Verify that the user has been deleted
-        get_response = self.client.get(user_uri)
-        self.assert_response_status(get_response, status.HTTP_404_NOT_FOUND)
+        try:
+            get_response = User.objects.get(username=test_user.username)
```

Comment:
Don't use the variable name `get_response` here because it is not a response object that is being returned, it is a `User` object. Using the word `response` here is confusing.